### ISSUE TITLE
This repo isn't in weekly trends

### ISSUE THREAD

**Issue by XakepSDK** [upend/IF_MS_BUYS_GITHUB_IMMA_OUT#88]
Just look at this: 3400++ starts in 4 days and this repo now isn't in trending list.
![a](https://user-images.githubusercontent.com/4938483/41021429-3b55442c-6988-11e8-90a4-469645c008ba.png)
Micrototalitarism from microsoft.

**Comment by Bloggerschmidt** [comment]
This is called censorship.

**Comment by zskovacs** [comment]
No, this is called paranoia...

![image](https://user-images.githubusercontent.com/20083522/41021688-99164b18-6967-11e8-82e2-37f06f48c1b2.png)

**Comment by XakepSDK** [comment]
Wtf, i can't find it

**Comment by XakepSDK** [comment]
It is found in `https://github.com/explore`, but not in https://github.com/trending?since=weekly

**Comment by vassudanagunta** [comment]
They clearly had something to take it out. [Because this was rather embarrassing](https://twitter.com/UpEnd_org/status/1003936552840515584).

**Comment by Omeryl** [comment]
The trending repos work in funny ways, and it's been known to drop repos that are consistently popular some times to let others shine.

**Comment by vassudanagunta** [comment]
@Omeryl That isn't likely because `ry / deno` has been on that board before this group, and was along side us this entire time.

**Comment by Omeryl** [comment]
@vassudanagunta It's also been known to remove repositories that are abusive in fashion, like your insistent need to edit comments to call peoples trolls. So perhaps that's biting you now.

**Comment by vassudanagunta** [comment]
@Omeryl it's not the same thing. 

This org isn't an open, neutral forum. It is one with a specific mission, a specific agenda. People who disagree with it have plenty of places to voice their opinion. They don't get to derail ours with a misused notion of free speech.

GitHub purports to be an open, neutral place. Not the same thing. 

The ACLU fights for people's civil liberties, and will even defend the right of the KKK to speak and assemble (peacefully). I applaud that.  But does the ACLU let KKK members into its meetings who want to criticize the ACLU's work for black civil rights?  Of course not. 

Would it have been a contradiction for Martin Luther King, John Lewis and other leaders of the non-violent civil rights movement of the 60s to disallow the participation of the KKK (white supremacists) or the Black Panthers or the Nation of Islam (black rights groups advocating violence) in their meetings and protests? Of course not. Even though their movement was about civil rights such as free speech for everyone.

Too many people repeat this mistake of misunderstanding the notion of "free speech". 

GitHub purports to be an open, neutral place. But presumably now you can't develop a GitHub competitor or anything that will threaten Microsoft's business, profits or dominance.  We all expected this. It's precisely why this effort exists and so many people support it.

**Comment by Omeryl** [comment]
@vassudanagunta Perhaps. Then just close the issue, lock it, and call it a day. You took it a step further by abusing a feature to edit the issues to make the users call themselves a troll. That's entirely different.

**Comment by vassudanagunta** [comment]
@Omeryl 

> insistent need to edit comments to call people trolls

Because the posts made were troll posts, or disrespected the rules for posting. You are welcome to go to all those troll posts, press the edit button to look at their original messages. The original titles are also right there in the thread.

**Comment by vassudanagunta** [comment]
@Omeryl I'm not going to debate you on this anymore. There are too many things to do. Can't afford to grapple with trolls and those who defend them. Sorry.

**Comment by nguyenkien** [comment]
If you don't like Microsoft/Github, just leave.

**Comment by e-kwsm** [comment]
You should not censor comments.

**Comment by ahsanulhaque** [comment]
Grow up. Github is a private entity. They can do whatever they want. If you want a free and open space with flowers and sunshine, then go set up your own.

**Comment by nukeop** [comment]
Already on HN: https://news.ycombinator.com/item?id=17245018

**Comment by vassudanagunta** [comment]
This group is like a big meeting. This meeting has a specific agenda. It is not a "free speech" meeting. There are plenty of places for free speech. 

Debates about which GitHub alternative, or about [whether an image I posted was insensitive to women are on topic](https://github.com/upend/IF_MS_BUYS_GITHUB_IMMA_OUT/pull/58). Posts by people opposed to this group or its aims don't belong here and their only purpose is to undermine this group. They can do that anywhere else but here.

I've only had 3 hours of sleep for four nights in a row, and zero so far tonight (4:30am). I'm doing the best I can to keep this project on track.

**Comment by nukeop** [comment]
The "insensitive to women" thing is a classic derailment tactic employed by trolls. You should not have responded to it, or yielded to their demands.

**Comment by vassudanagunta** [comment]
I just updated the homepage (README) with the news about the Microsoft censorship.

**Comment by e-kwsm** [comment]
You justify your censorship but abuse MS's?

**Comment by vassudanagunta** [comment]
To those of you unhappy with my censoring comments and people who come here to undermine the project, or who demand that this be a "total free speech" zone, will see this as contradictory or hypocritical.

GitHub purports to be an open, neutral place, for all projects (within their terms of service). This group is not. 

This org isn't an open-ended, neutral forum. It is one with a specific mission, a specific agenda. People who disagree with it have plenty of places to voice their opinion. They don't get to derail ours with a misguided notion of total free speech.

The ACLU fights for people's civil liberties, and will even defend the right of the KKK to speak and assemble (peacefully). I applaud that. But does the ACLU let KKK members into its meetings who want to criticize the ACLU's work for black civil rights? Of course not.

Would it have been a contradiction for Martin Luther King, John Lewis and other leaders of the non-violent civil rights movement of the 60s to disallow the participation of the KKK (white supremacists) or the Black Panthers or the Nation of Islam (black rights groups advocating violence) in their meetings and protests? Of course not. Even though their movement was about civil rights such as free speech for everyone.

GitHub purports to be an open, neutral place. But presumably now you can't develop a GitHub competitor or anything that will threaten Microsoft's business, profits or dominance. We all expected this. It's precisely why this effort exists and so many people support it.

**Comment by vassudanagunta** [comment]
@e-kwsm, we will have to disagree. If you can't expect this decision of mine, I ask you to leave the group. Please stop debating this here. Feel free to create your own group if you'd like.

**Comment by Droppers** [comment]
You are abusing other peoples opinion by editing and removing comments that don't fit your opinion.

**Comment by MoonlightOwl** [comment]
Without any moderation at all, this repo will turn into a pile of shit very fast.

**Comment by RyanGannon** [comment]
@Droppers This repo has a clear purpose and it makes sense to remove threads that aren't relevant to that. It doesn't stop you from finding an alternative or using both. That is the freedom we have on this platform, to provide an opportunity for all views but requiring all people to accept them.

**Comment by vassudanagunta** [comment]
@Droppers No, they are posting graffiti (which I normally love), and I am power washing it away. Anyone at any time can see their original comment. It's there for the looking. 

Please stop debating this. This org has a leader. I've worked extremely hard to get us here. I can't please everyone in this group and also keep it focused. There are thousands of people coming here to see what we're doing, and all this noise, both the trolls and this debate, undermines the mission of this org that I started. Please respect me as the creator. You are welcome to leave, but I'd so much rather you gave me a chance and stayed.

**Comment by Droppers** [comment]
@RyanGannon I agree with that, but removing or editing any counter-argument is not helping anyone.

**Comment by vassudanagunta** [comment]
@Droppers I'm removing noise. I'm removing meeting interruptions that are off-topic. People come here to read what is being done re the mission. But if they come and see a bunch of debates like this, they's decide we don't have a chance and leave. 

This is my last reply on this. I need to get back to the mission. Please stop.

**Comment by thrasos** [comment]
And there goes Github's sense of humour (among other things).

**Comment by vassudanagunta** [comment]
@Omeryl, @e-kwsm, @Droppers and anyone else who dislikes my censorship. I created a space for you guys to express your disagreement: #91.

**If you guys self-delete your comments about censorship from here and other places** and promise to keep them within #91, then you can have that forum. Deal? That's the best I can do.

**Comment by Omeryl** [comment]
@vassudanagunta Thanks for finally unbanning me from talking here! That's a great relief, I was annoyed that you only censored some of my comments and not all of them, and being banned made it impossible to remove my out-of-context comments. 

It's a great step forward, but I'm going to leave because it's obvious that you refuse to listen to any reasoning from the "enemy." Thanks for the too little too late. 

For anybody that is curious, here's the uncensored version of this issue: https://web.archive.org/web/20180606080036/https://github.com/upend/IF_MS_BUYS_GITHUB_IMMA_OUT/issues/88

**Comment by Omeryl** [comment]
@vassudanagunta Before I go, if your sleep situation is that dire, you really need to take a break and focus on yourself. This issue really shouldn't be negatively affecting you as much as it seems to have. You first, corporate purchasing decisions second.

**Comment by HExiangyu** [comment]
前排兜售香烟瓜子火腿肠啤酒饮料

**Comment by quantumpacket** [comment]
Not showing up for me anymore. I came and looked just for this thread.

**Comment by jorgebucaran** [comment]
I am far from an expert, but I think is very plausible this project falls under the gray area of GitHub TOS ([see 2 and 3 of acceptable use](https://help.github.com/articles/github-terms-of-service/#c-acceptable-use)). So, wouldn't be reasonable to expect someone at GitHub simply removed it from the github.com/trending as a silent warning?

**Comment by kuharan** [comment]
Also, check out the developer Ranks. How is Nadela Ranked as No.1?  I mean, WTF?

![image](https://user-images.githubusercontent.com/22185333/41058167-b7c8be68-69e6-11e8-9cdd-61e611cabea9.png)

**Comment by jorgebucaran** [comment]
@kuharan I think the only reason he's showing up there is that a lot of users happen to be following his account. Not at all odd considering recent events. It's just an algorithm.

**Comment by hktonylee** [comment]
This reminds me [https://www.theregister.co.uk/AMP/2016/04/15/microsoft_discontinues_robovm/]. The M$ guy who discontinued a rival open source project is exactly the same guy in charge of Github.

**Comment by vassudanagunta** [comment]
Hey all. I love the work you've been doing. Not everyone loves mine. Please read Issue 114.

**Comment by vassudanagunta** [comment]
And here's traffic to the repo. Their censorship did exactly what they wanted, but not entirely.

![traffic](https://user-images.githubusercontent.com/2830093/41169508-384e7b82-6b17-11e8-919a-37b4bfa2daff.png)

**Comment by Atavic** [comment]
I like how this issue is evolving: a ground-up initiative is speaking loud towards the big MS elephant who entered the room full of containers of Open Source repos.

**Comment by vassudanagunta** [comment]
reminder this forum is shutting down in 5 hours 20 minutes.

**Comment by vassudanagunta** [comment]
Hello,

The GitHub Evacuation Project has moved to GitLab (not an endorsement or even a final home). Your enthusiasm and contribution is still needed. Please check out [the new project home](https://gitlab.com/upend/github), and read [the project wiki](https://gitlab.com/upend/github/evac-guide/wikis) for info to get restarted.

Thank you!

vas
